packages/react-charting/src/components/AreaChart/AreaChart.types.ts

```diff
@@ -53,6 +53,12 @@ export interface IAreaChartProps extends ICartesianChartProps {
    * Optimize area chart rendering for large data set.
    */
   optimizeLargeData?: boolean;
+
+  /**
+   * @default false
+   * The prop used to enable the perf optimization
+   */
+  enablePerfOptimization?: boolean;
```

Comment:
This change should go away if you merge master.

Reply:
Done